Vagrantfile

```diff
@@ -18,16 +18,14 @@ config = YAML.load_file(config_file)
 Vagrant::Hosts::check_for_ssh_keys
 
 ### take BOX from the enviroment ###
-BOX = ENV.has_key?('BOX') ? ENV['BOX'] : 'opensuse/openSUSE-42.3-x86_64'
+BOX = ENV.has_key?('BOX') ? ENV['BOX'] : 'opensuse/openSUSE-15.1-x86_64'
```

Comment:
there is no `opensuse/openSUSE-15.1-x86_64` in config.yml

Reply:
fix (not sure why this config has this strange `sub0/XXX` name)

Reply:
that shouldn't be sub0, @colder-is-better would fix it soon.